scripts/verify-release-artifacts.sh

```diff
@@ -18,6 +18,10 @@
 
 set -o pipefail
 
+cur_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+soci_snapshotter_project_root="$(cd -- "$CUR_DIR"/.. && pwd)"
```

Comment:
nit: can we just use `"$CUR_DIR"/..` like we do in other files? e.g. https://github.com/awslabs/soci-snapshotter/blob/db7df3ab2a840bc927417a84448263885b3e21ff/scripts/add-ltag.sh#L20

Also, I think you have a bug because `CUR_DIR` is capitalized here, but not in the declaration above. 

I think most of our scripts are using the wrong convention and we should be using lower case letters. https://stackoverflow.com/questions/673055/correct-bash-and-shell-script-variable-capitalization

Reply:
I'll try and fix that for all of our scripts at some point, but for now fixed for this

Reply:
Oops, I remember why I did this. It makes the path funny because it'll end up making the release directory something like `~/soci-snapshotter/scripts/../release`. Fixed capitalization bug though.